### PR TITLE
update for v1.119 on 5/2/18

### PR DESCRIPTION
**Pull request by VividWombat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +63,8 @@ payload = [OrderedDict([
 encoded = json.dumps(payload).replace(' ', '')
 
 user_key = "thisistheuserskey"
-# As of 12:00 28/08/2017 UTC (version 1.108 / timestamp 1503565657)
-secret = "yOy3qr/HW9NZ9iLXjYLVADMO7wKMZcTgsUVqcqkl+h7ddVER8sHYEH6bxsSJOzerXci2kJKcMM9xQZjmdVD08Q=="
+# As of 11:12 05/02/2019 UTC (version 1.119 / BUILD_NUMBER:String = "7cb313c7be")
```
Comment (m3talstorm) [inline README.md]: Don't think its 2019 yet ;)
Can we keep the timestamp there

File: README.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ Used by [FoE Bot](https://github.com/m3talstorm/foe-bot) for request signature g
 *Purely educational and to fuel curiosity*
 
 ## Summary
-- Binary: https://foeen.innogamescdn.com/swf/Main.swf?1511779469
-- Version: 1.116
-- Timestamp: 1513162258
-- Secret: Ob90uMKvbFGtB7/yWM19s51P/hOnbg289yT4V7qHTs8qTr/vefAMsYesk21tRIxi7Gx/9byISR/a3nqz53QuyQ==
+- Binary: https://foeen.innogamescdn.com/swf/Main.swf
+- Version: 1.119
+- Timestamp: 05.02.2018 11:12
```
Comment (m3talstorm) [inline README.md:12]: Can we keep the timestamp in the same format, it makes it easier to marry up requests.

**Review by m3talstorm — CHANGES_REQUESTED** [review]
Add review comments

File: README.md (hunk 2 above)
Comment (VividWombat) [reply README.md:12]: I copied the timestamp from  the swf. dont think it has it in unix format, but will check

**Pull request closed without merging**